proposals/0233-codepoint-and-character-literals.md

```diff
@@ -0,0 +1,319 @@
+* Proposal: SE-0233
```

Comment:
When you revise this, can you add the title of the proposal as an H1 here?

Reply:
done